cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -2926,6 +2965,32 @@ instance Key ByronKey where
     verificationKeyHash (ByronVerificationKey vkey) =
       ByronKeyHash (Byron.hashKey vkey)
 
+
+instance SerialiseAsRawBytes (VerificationKey ByronKey) where
+    serialiseToRawBytes (ByronVerificationKey (Byron.VerificationKey xvk)) =
+      Crypto.HD.unXPub xvk
+
+    deserialiseFromRawBytes (AsVerificationKey AsByronKey) bs =
+      either (const Nothing) (Just . ByronVerificationKey . Byron.VerificationKey)
+             (Crypto.HD.xpub bs)
+
+instance SerialiseAsRawBytes (SigningKey ByronKey) where
+    serialiseToRawBytes (ByronSigningKey (Byron.SigningKey xsk)) =
+      Crypto.HD.unXPrv xsk
+
+    deserialiseFromRawBytes (AsSigningKey AsByronKey) bs =
+      either (const Nothing) (Just . ByronSigningKey . Byron.SigningKey)
+             (Crypto.HD.xprv bs)
+
+instance SerialiseAsBech32 (VerificationKey ByronKey) where
+    bech32PrefixFor         _ =  "addr_xvk"
```

Comment:
what does the x stand for here?

Reply:
eXtended

See https://github.com/cardano-foundation/CIPs/pull/6